packages/api-client-core/src/InternalModelManager.ts

```diff
@@ -239,11 +239,35 @@ export class InternalModelManager {
   constructor(
     private readonly apiIdentifier: string,
     readonly connection: GadgetConnection,
-    readonly options?: { pluralApiIdentifier: string }
+    readonly options?: { pluralApiIdentifier: string; hasAmbiguousIdentifiers: boolean }
```

Comment:
This is why this change won't break when new clients are generated that use old api-client-core versions right?

Reply:
yep!